src/desktop/lib/admin_only.coffee

```diff
@@ -1,7 +0,0 @@
-module.exports = (req, res, next) ->
-  if req.user?.get('type') isnt 'Admin'
-    err = new Error 'You must be logged in as an admin'
-    err.status = 403
-    next err
-  else
-    next()
```

Comment:
Are no other apps relying on `admin_only.coffee` imports? (If we drop the extension it should just work)

Reply:
I made changes above-- there were ~3 other places that referenced it with the extension